### PR TITLE
Organize technical documentation and onboarding(contributing) documentation for project

### LINKED ISSUE #25: Organize technical documentation and onboarding(contributing) documentation for project

### PR DESCRIPTION
**Pull request by leekahung**
Just getting the organization process started by:

1. Moving README.md (React technical documentation) to /docs directory
2. Renaming PASS_Summary.md to README.md
3. Renaming onboarding.md to CONTRIBUTING.md

Additional feedback to file updates/project structure are more than welcomed before merging with Development branch and before merging Development with master branch as discussed at Feb 21 meeting. Things such as:

- Updates to PASS_Summary.md (Renamed README.md) to reflect current status
- Updates to onboarding.md (Renamed CONTRIBUTING.md) to reflect current status
- Directory structure
- Establishing rules for commits/push onto Development branch
- Etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,78 @@
...
+| ------------------- | --------------------- | -----------| ------------------------|
+| Team Lead - Kevin M.| Team Lead - Jared K.  | Gabby      | Flo                     |
+|                     |                       | Kyle       | Danica                  |
+|                     |                       |            | Wilfred                 |
+|                     |                       |            | Katherine               |
+|                     |      
+
+
+
+## Check out the documentation:
```
Comment (GoodGuyGregory) [inline README.md:73]: Could we change this to something like **Additional Resources** or something like **Additional Reading**? 🤔

File: README.md [hunk 2]
```diff
@@ -1,53 +1,78 @@
-# PASS
+# PASS - Personal Access System for Services
```
Comment (GoodGuyGregory) [inline README.md:1]: @leekahung Overall this looks great! thanks for moving it into the main page

File: README.md (hunk 1 above)
Comment (leekahung) [reply README.md:73]: Totally! I think that's a good idea.

**Review by GoodGuyGregory — APPROVED** [review]
@leekahung Overall this is fine with me. I can't speak for the whole team but I would Approve this merge

File: README.md (hunk 1 above)
Comment (GoodGuyGregory) [reply README.md:73]: If you want to make that swap you can after the merge. its up to you. I went  ahead and marked everything as good to go 👍

**Comment by GoodGuyGregory** [comment]
@leekahung Thanks for the PR

**Pull request merged**